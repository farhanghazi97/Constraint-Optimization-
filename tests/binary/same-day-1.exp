m1:fri 9am
m2:thu 9am
m3:fri 9am
m4:thu 9am
m5:wed 9am
m6:tue 9am
m7:tue 9am
m8:wed 9am
m9:mon 9am
m10:mon 9am
cost:0
