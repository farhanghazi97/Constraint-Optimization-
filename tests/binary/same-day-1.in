meeting, m1
meeting, m2
meeting, m3
meeting, m4
meeting, m5
meeting, m6
meeting, m7
meeting, m8
meeting, m9
meeting, m10
constraint, m1 same-day m3
constraint, m2 same-day m4
constraint, m8 same-day m5
constraint, m6 same-day m7
constraint, m10 same-day m9
domain, m3, fri, hard
domain, m4, thu, hard
domain, m8, wed, hard
domain, m7, tue, hard
domain, m10, mon, hard
