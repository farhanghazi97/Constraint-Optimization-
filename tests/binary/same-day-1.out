m1:fri 12pm
m2:thu 9am
m3:fri 12pm
m4:thu 9am
m5:wed 1pm
m6:tue 12pm
m7:tue 12pm
m8:wed 1pm
m9:mon 1pm
m10:mon 1pm
cost:0
